src/routes/Tool/Competences.tsx

```diff
@@ -159,17 +129,26 @@ const Competences = () => {
         </div>
 
         <div className="order-3 col-span-2 sm:order-2 sm:col-span-4">
-          <InputField label="Taitosi" value={skill} onChange={(event) => inputChangeHandler(event.target.value)} />
-          <div className="mt-7 min-h-[200px] rounded border p-4">
-            <ul className="flex flex-col space-y-4">
-              {competences.map((item) => (
-                <li className="flex flex-row justify-between rounded border p-4" key={item.id}>
-                  <span>{item.nimi}</span>
-                  <span>{item.osuvuus}</span>
-                </li>
-              ))}
-            </ul>
-          </div>
+          {!isLoading ? (
+            <FormProvider {...methods}>
+              <Form>
+                <InputField
+                  label={t('work-history.job-duties')}
+                  value={tyotehtava}
+                  onChange={(event: ChangeEvent<HTMLInputElement>) => setTyotehtava(event.target.value)}
+                  placeholder="Lorem ipsum dolor sit amet"
+                  help="Help text"
+                />
```

Comment:
```suggestion
                <div className="mb-6">
                  <InputField
                    label={t('work-history.job-duties')}
                    value={tyotehtava}
                    onChange={(event: ChangeEvent<HTMLInputElement>) => setTyotehtava(event.target.value)}
                    placeholder="Lorem ipsum dolor sit amet"
                    help="Help text"
                  />
                </div>
```

Reply:
It seems that in the design the margin I missed it is `16px`, so it should instead be `mb-5`